### DISCUSSION TITLE
BthPS3 stops working after more than 9 paired (not connected!) devices

### DISCUSSION THREAD

**Discussion by hoderer** [nefarius/BthPS3#69] [Support]
First of all: I'm SO HAPPY to put some new life into my old DualShock3-Gamepads :)

But I ran into a strange problem with BthPS3: I got 9 different gamepads paired (not connected, just paired) and when I pair a 10th one, BthPS3 stops working. No matter what gamepad I pair, it's always the same: 9 ok for BthPS3, 10 not. If I delete one from the paired devices list and restart, everything is working again. I don't think it's a bluetooth problem, since you can pair as may devices as you like and connect about 7 or so.

My Bluetooth-Controller is the Intel internal one, which is on the supported devices list and I'm using the latest official version of BthPS3 (the one from august 2022).

Can anyone reproduce this? If not, I post some logs of course. Thanks for your help!

**Comment by nefarius** [comment]
Intel Wireless has always been "special", I wouldn't trust it further than I can throw it. I recommend you try a different, dedicated USB dongle instead.

**Reply by hoderer** [reply]
I did a quick test with the ASUS BT500 (which, as they say, is one of the best bt-dongles out there at the moment).

1. I deleted all the old pairings
2. I deactivated Intel Bluetooth in the device manager
3. I deinstalled BthPS3
4. I plugged in my BT500
5. Installed latest drivers
6. I installed BthPS3

Everything is working fine. My DualShock3-Controllers are perfectly connecting. Then I paired 9 of my controllers (mostly from 8bitdo) and Dualshock3-Controllers are still connecting fine. Then I paired another two 8bitdo-controllers and BthPS3 stopped working after a restart. No matter what I choose in the configuration tool, nothing is working as designed anymore. When I delete the last two pairings, everything is good again. It's the exact same thing with the intel bluetooth device.

Is there any way I can help debugging this? I couldn't find any logs in the install directory. THANKS alot and again, thanks for your great work!!!

**Reply by nefarius** [reply]
[Follow this](https://vigem.org/projects/BthPS3/Debugging-the-drivers/) while performing the problematic actions and I'll take a look if we see anything in the driver logs.

**Reply by hoderer** [reply]
Thanks alot for your support and fast reply. I try as fast as I can.

**Reply by nefarius** [reply]
Btw. you do not need to reinstall BthPS3 when changing Bluetooth hosts.

**Reply by hoderer** [reply]
Here it is. I paired the 10th controller and it won't connect anymore. Thats with the intel one, because after I failed with BT500 (which I borrowed from a friend) I used acronis true image to get back to my old installation, but since it's the same behavior it should not make any difference. If needed, I of course will post logs with BT500.

[BthPS3.zip](https://github.com/ViGEm/BthPS3/files/10798439/BthPS3.zip)

**Reply by nefarius** [reply]
Thank you, found the issue 😁 Gonna need to ship an update 🙃 

![image](https://user-images.githubusercontent.com/286631/220582168-30e41ae2-40d4-43fe-a59d-c58533d6bab3.png)

**Reply by hoderer** [reply]
Great News! Can't wait to try it out! Thanks for looking into it.

**Reply by hoderer** [reply]
Please don't get me wrong, I know developers hate things like that, but I'm just curious. Is there a planned release date or does it take longer? You can - of course - delete this post and REALLY (!) sorry if that question bothers you.

**Reply by nefarius** [reply]
There isn't as business tasks, friends and family matters take priority and then when some day has a few spare hours I look into hobby projects like these. Have done so since 2014 and been very successful. So if you don't see any update from me, rest assured there isn't anything worth reporting going on. A release will appear at the usual location when it's done, as simple as that.

**Reply by nefarius** [reply]
And what point would it be in deleting posts, your post already caused me to read it and therefore invest time in a response so if you seek helping me, you help best by keeping my inbox quiet 🙂
